teps/0096-pipelines-v1-api.md

```diff
@@ -123,19 +131,59 @@ of TEP-0033: Feature Gates, Tekton's policy allows a behavior change to be updat
 A behavior can't be changed from opt-in to opt-out until 9 months after it is introduced to a beta API or 12 months after it is introduced to a stable API.
 This is different from Kubernetes' policy, which would require creating a new API version to change a feature from opt-in to opt-out.
 
-### API Definition
-
 ## Use Cases
 
-### V1 Supported Use Cases
+Most Tekton use cases are supported by features from multiple Tekton projects, meaning that there are relatively few use cases for
+Pipelines as an independent project. The following use cases guide Tekton feature work, and a Pipelines V1 API should stabilize
+the features critical to supporting them.
+
+### Use Cases prioritized for Tekton
```

Comment:
this is a great start, thank your for adding these!

Reply:
this is a limited set of use cases but a great start, do we envision adding more in next iterations? 

I would start this paragraph with:

Tekton is a cloud-native CI/CD platform and is designed to support modern CI/CD use cases, some of them are listed here:

Reply:
Reworded to make it more clear that this isn't intended to be an exhaustive list of use cases.